docs/bigtable-client.rst

```diff
@@ -1,6 +1,12 @@
 Client
 ~~~~~~
 
+.. warning::
+
+    gRPC is required for using the Cloud Bigtable API. As of May 2016,
```

Comment:
I intentionally am putting the same banner on every page, so if people land on any of the docs pages (from links or search results), they will get the warning.

Are you asking me to include an RST template?